src/stacks-hierarchy/Root_NotificationPermissionScreen.tsx

```diff
@@ -0,0 +1,103 @@
+import {ScrollView, useWindowDimensions, View} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import {useTranslation} from '@atb/translations';
+import {Button} from '@atb/components/button';
+import React from 'react';
+import {themeColor} from '@atb/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_WelcomeScreen';
+import {StyleSheet} from '@atb/theme';
+import {useFocusOnLoad} from '@atb/utils/use-focus-on-load';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import NotificationPermissionTexts from '@atb/translations/screens/NotificationPermission';
+import {PushNotification} from '@atb/assets/svg/color/images';
+import {RootStackScreenProps} from '@atb/stacks-hierarchy/navigation-types';
+import {useAppState} from '@atb/AppContext';
+
+type Props = RootStackScreenProps<'Root_NotificationPermissionScreen'>;
+
+export const Root_NotificationPermissionScreen = ({navigation}: Props) => {
+  const {t} = useTranslation();
+  const styles = useThemeStyles();
+  const focusRef = useFocusOnLoad(true, 200);
+
+  const {completeNotificationPermissionOnboarding} = useAppState();
+
+  const {width: windowWidth} = useWindowDimensions();
+
+  const handleButtonPress = () => {
+    // TODO: add permission logic
+    completeNotificationPermissionOnboarding();
+    navigation.popToTop();
+  };
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView contentContainerStyle={styles.contentContainer}>
+        <View style={styles.mainView} ref={focusRef} accessible={true}>
+          <PushNotification width={windowWidth} />
```

Comment:
Given the shape of the image and its positioning, I think it is better to set the size using height in this case